__tests__/app.test.js

```diff
@@ -483,3 +483,37 @@ describe("/api/users", () => {
     });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  describe("DELETE", () => {
+    it("204: deletes the comment with the given id", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(() => {
+          return db.query(`SELECT * FROM comments`).then(({ rows }) => {
```

Comment:
This way of checking is fine, but remember you could use a WHERE clause here to look for the comment_id = 1 and see that it is an empty array.